requirements-dev.in

```diff
@@ -0,0 +1,5 @@
+-r demo-code/requirements.txt
+pytest
+pytest-django
```

Comment:
I think you need coverage and pytest-cov for the options in pyproject.toml to work